google/resource_google_project.go

```diff
@@ -210,6 +210,17 @@ func resourceGoogleProjectCheckPreRequisites(config *Config, d *schema.ResourceD
 	if !stringInSlice(resp.Permissions, perm) {
 		return fmt.Errorf("missing permission on %q: %v", ba, perm)
 	}
+	errExpected := "googleapi: Error 403: Project '00000000000' not found or permission denied., forbidden"
+	errAPINotEnabled := "accessNotConfigured"
```

Comment:
nit: inlining this seems fine

Reply:
Okay. I thought it would make it easier to read, but I was wrong. Going to revert the commit.